common/utils/EncryptionManager.js

```diff
@@ -16,6 +17,10 @@ class EncryptionManager {
     return decipher.update(text, CONST.ENCRYPTION.OUTPUT_ENCODING, CONST.ENCRYPTION.INPUT_ENCODING);
   }
 
+  async generateSshKeyPair(tempUser) { //jshint ignore: line
+    const sshKeyGenerator = new RsaKeyGenerator(tempUser);
```

Comment:
Can you please keep the class name and file name same? `SshRsaKeyGenerator` vs `RsaKeyGenerator`